woodwork/exceptions.py

```diff
@@ -13,6 +13,12 @@ def get_warning_message(self, standard_tags_to_remove, name):
         return f"Removing standard semantic tag(s) '{', '.join(standard_tags_to_remove)}' from column '{name}'"
 
 
+class StandardTagsChangedWarning(UserWarning):
```

Comment:
Do we still need the `StandardTagsRemovalWarning` class above this now that we are using this new class or do we need to leave it until we delete DataColumn/DataTable?

Reply:
We do currently need it for DataColumn. I think it'd be better to leave it until it's no longer being used.